email-templates/addedModerator.html

```diff
@@ -0,0 +1,413 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
I dont exactly know all the things we want to list in this email.
Would be nice to get some input.

Reply:
How about, loosely:

```
You have been added as a moderator in the ${community.name} community on Spectrum! Moderators in this community can:
- lock conversations to prevent future messages
- delete conversations that are not relevant or negative for the community
- delete messages that aren't productive to conversations
- block people in the community who are being disruptive
- manage community and channel settings
- view community analytics (if enabled) to better understand the people and content in a community

[ Go to the ${community.name} community ]
[ View community settings ]
```